dev-libs/libebml/patches/libebml-1.4.4.patchset

```diff
@@ -0,0 +1,34 @@
+From 9f0839f6e9abdaf1533ae5f0e2752ce40f5be487 Mon Sep 17 00:00:00 2001
+From: robxnano <89391914+robxnano@users.noreply.github.com>
+Date: Sun, 12 Feb 2023 15:08:24 +0000
+Subject: [PATCH] Haiku: Only include SupportDefs.h on x86_gcc2
+
+---
+ ebml/c/libebml_t.h | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/ebml/c/libebml_t.h b/ebml/c/libebml_t.h
+index b124369..a8853f9 100644
+--- a/ebml/c/libebml_t.h
++++ b/ebml/c/libebml_t.h
+@@ -72,7 +72,7 @@ extern "C" {
+     typedef uint16_t uint16;
+     typedef uint8_t uint8;
+ # endif // __GNUC__
+-#elif defined(__BEOS__) || defined(__HAIKU__)
++#elif defined(__BEOS__) || (defined(__HAIKU__) && __GNUC__ == 2)
```

Comment:
Getting errors on buildmaster (and local) for gerbera, could this be related?
https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/4826/builds/81325.log